felix/bpf-gpl/Makefile

```diff
@@ -14,7 +14,8 @@ CFLAGS +=  \
 	-O2 \
 	-target bpf \
 	-emit-llvm \
-	-g
+	-g \
+	-D__x86_64__
```

Comment:
we would not need this if we upgraded our linux headers and should not break arm64 either. Just fixes importing <linux/icmpv6.h>

Reply:
Not sure about arm64. Should we try it out?